.woodpecker/test.yaml

```diff
@@ -141,18 +143,18 @@ steps:
     failure: ignore
 
 services:
-  sv-postgres:
+  postgres:
     image: docker.io/postgres:16
     ports: ['5432']
     environment:
-      - POSTGRES_USER=postgres
-      - POSTGRES_HOST_AUTH_METHOD=trust
+      POSTGRES_USER: postgres
+      POSTGRES_HOST_AUTH_METHOD: trust
     when: *when
 
-  sv-mysql:
+  mysql:
```

Comment:
This reverts the step name to temp fix the duplicate names.

Reply:
ah yes ... that was picked form my old pull and did not adjust to the current conf ...